client/sites-dashboard/components/sites-dashboard.tsx

```diff
@@ -60,7 +60,7 @@ const DashboardHeading = styled.h1`
 
 export function SitesDashboard( { launchStatus }: SitesDashboardProps ) {
 	const { __ } = useI18n();
-	const sites = useSelector( getSites );
+	const sites = Object.values( useSelector( getSiteExcerpts ) );
```

Comment:
Still need to sort these sites, and put the primary site / most recent sites on top.